cache/manager_test.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 func TestManager(t *testing.T) {
+	t.Parallel()
 	ctx := namespaces.WithNamespace(context.Background(), "buildkit-test")
```

Comment:
Should we randomize the namespace name as in tmpdir?
Future tests may conflict for `assert(expectedSnapshotList, listSnapshot(ctx, ...))`

Reply:
I'm not sure how if they use different state directories. There is no actual containerd running in this test. We should also look into removing these namespaces as they shouldn't be needed at this level(I'm not sure if they actually are needed anymore with newer containerd versions but need to double check).